go/tools/gazelle/gazelle/main.go

```diff
@@ -145,23 +151,26 @@ func main() {
 	}
 }
 
-func readBuild(root string) (data []byte, location string, _ error) {
-	for n := range validBuildNames {
-		p := filepath.Join(root, n)
-		b, err := ioutil.ReadFile(p)
+func findBuildFile(repo string) (string, error) {
+	for _, base := range buildFileNames {
+		p := filepath.Join(repo, base)
+		fi, err := os.Stat(p)
 		if err != nil {
-			if os.IsNotExist(err) {
-				continue
-			}
-			return nil, "", err
+			continue
```

Comment:
If the stat fails for a reason other than os.IsNotExist, isn't that a sign of bad things?